src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -99,6 +99,7 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
     <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0" />
+    <Reference Include="System.Net.Http" />
```

Comment:
by @vincentkam 
Hm, now I'm slightly confused: According to this, HttpClient is available for the following platforms

>.NET Framework
4.8 4.7.2 4.7.1 4.7 4.6.2 4.6.1 4.6 4.5.2 4.5.1 4.5
.NET Standard
2.1 2.0 1.6 1.4 1.3 1.2 1.1

It seems weird for it to be available .Net Standard 1.4 and 1.6 but not 1.5, so this feels like a documentation error to me to be honest. I've confirmed via local testing that the current Driver.Core project builds when targeting .Net Standard 1.5 and 2.0, and fails to build when targeting .NET Framework 4.5.2 when I remove line 102 <Reference Include="System.Net.Http" />

My (other) question is: should we err on the side of safety and use a NuGet package reference instead of a regular reference? i.e.
<PackageReference Include="System.Net.Http" Version="4.3.4" /> (i.e. including https://www.nuget.org/packages/System.Net.Http/).

Reply:
by @vincentkam 
I think it may be safer to use the PackageReference. I believe by using Reference, the application will need to have the dll in its path somewhere: for our developer machines and the evergren machines, that wouldn't be a problem because we have the DLL from later versions of .NET. By using a PackageReference, I think that it'd allow a machine to not have the DLL on its path at all, as it'll simply download the package from Nuget as needed. What do you think Mikalai?

Reply:
I'm not really familiar with the differences between these two cases. But just to note, we use PackageReference  in other places in this file